pkg/cmd/controller/pipelinerunner.go

```diff
@@ -457,3 +464,42 @@ func (o *PipelineRunnerOptions) setupSignalChannel(cancel context.CancelFunc) {
 		cancel()
 	}()
 }
+
+func (o *PipelineRunnerOptions) getSourceURL(org, repo string) string {
+	jxClient, ns, err := o.getClientsAndNamespace()
```

Comment:
You are missing an err check here. _jxClient_ might be nil here.

Reply:
done